src/components/CardContainer/CardContainer.tsx

```diff
@@ -5,22 +5,25 @@ import { ICard } from "../../Utils/Types";
 import { API_KEY } from "../../store/Utils/storeConstances"
 import { fetchFromApi } from "../../store/Utils/index";
 import { useAppDispatch, useAppSelector } from "../../store/store";
-import { EVERYTHING_STRING, TOP_HEADLINES_STRING } from "../../store/Utils/storeConstances"
+import { EVERYTHING_URL, TOP_HEADLINES_URL } from "../../store/Utils/storeConstances"
 
-const CardContainer = () => {
 
+const CardContainer = () => {
   const dispatch = useAppDispatch();
   
   useEffect(()=>{
-    dispatch(fetchFromApi(EVERYTHING_STRING + API_KEY));
+    dispatch(fetchFromApi(EVERYTHING_URL + API_KEY));
```

Comment:
I think it is better to store all strings in the same file

Reply:
all strings in the "storeConstances" file.
Is this what you meant?